README.md

```diff
@@ -13,7 +13,12 @@ cd /home/openfisca
 git clone git@github.com:openfisca/openfisca-ops.git
 ```
 
-Each sub-directory has its own README.md file.
+For more information on `auto-update` sub-directories, please see their own README.md files.
+
+## Server configuration
+
+Main nginx services' configuration files are stored in a directory named after a server name.  
+For example: `www.openfisca.fr` configuration file is `openfisca-ops/www.openfisca.fr/www.openfisca.fr.conf`
```

Comment:
For example: `www.openfisca.fr` 's configuration file is in `openfisca-ops/www.openfisca.fr/www.openfisca.fr.conf`

Reply:
's added. 
> www.openfisca.fr.conf is the configuration file itself.